src/Select/SelectInput.d.ts

```diff
@@ -10,7 +10,7 @@ export interface SelectInputProps extends StandardProps<{}, SelectInputClassKey>
   MenuProps?: Partial<MenuProps>;
   name?: string;
   onBlur?: React.FocusEventHandler<any>;
-  onChange?: (event: React.ChangeEvent<{}>, child: React.ReactNode) => void,
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>, child: React.ReactNode) => void,
```

Comment:
This one looks like it could be called on the select element, or on one of its children...